src/main/webapp/app/exercises/modeling/assess/modeling-assessment.component.ts

```diff
@@ -4,7 +4,9 @@ import { JhiAlertService } from 'ng-jhipster';
 import interact from 'interactjs';
 import { Feedback, FeedbackType } from 'app/entities/feedback.model';
 import * as $ from 'jquery';
+import { JhiLanguageService } from 'ng-jhipster';
```

Comment:
According to GitHub Actions / client-style, 'JhiLanguageService' is defined but never used. Should this be fixed?